tests/sha3.cc

```diff
@@ -70,6 +70,32 @@ TEST(ApiSuite, ApiTest)
 
     EXPECT_EQ(strncmp((char*)digest, (char*)expected_digest.data(), 32), 0);
   }
+
+  // Documentation.
+  // Lines after START and before END are used in documentation.
+  {
+    // ANCHOR(example shake128)
+    // This example uses SHAKE-128.
+    //
```

Comment:
Remove?

Reply:
?

Reply:
useless line

Reply:
Do you want to remove the trailing comment line? Or the line after it?